core/math/plane.h

```diff
@@ -67,49 +67,49 @@ class Plane {
 
 	/* misc */
 
-	Plane operator-() const { return Plane(-normal, -d); }
+	Plane operator-() const { return Plane(-normal, -distance); }
 	bool is_equal_approx(const Plane &p_plane) const;
 
 	_FORCE_INLINE_ bool operator==(const Plane &p_plane) const;
 	_FORCE_INLINE_ bool operator!=(const Plane &p_plane) const;
 	operator String() const;
 
 	_FORCE_INLINE_ Plane() :
-			d(0) {}
+			distance(0) {}
 	_FORCE_INLINE_ Plane(real_t p_a, real_t p_b, real_t p_c, real_t p_d) :
 			normal(p_a, p_b, p_c),
-			d(p_d) {}
+			distance(p_d) {}
```

Comment:
This is unrelated to your change, but a/b/c should be renamed to x/y/z at some point.

Reply:
Does that have to be done now? Because that will lead to another bunch of changes.

Reply:
No, it can be done later (in fact it's probably better if it's a separate PR).